webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala

```diff
@@ -518,22 +546,25 @@ class NodeApiService13 (
       runs <- reportsExecutionRepository.getNodesLastRun(nodes.keySet)
       n3 = System.currentTimeMillis
       _ = TimingDebugLoggerPure.trace(s"Getting run infos: ${n3 - n2}ms")
-      globalMode <- getGlobalMode()
+      (systemCompliances, userCompliances) <- reportingService.getUserAndSystemNodeStatusReports(Some(nodes.keySet))
       n4 = System.currentTimeMillis
-      _ = TimingDebugLoggerPure.trace(s"Getting global mode: ${n4 - n3}ms")
+      _ = TimingDebugLoggerPure.trace(s"Getting compliance infos: ${n4 - n3}ms")
```

Comment:
I don't think it should be that logger there, `TimingDebugLoggerPure` shouldn't do anything. You need to either call `TimingDebugLogger` or `TimingDebugLoggerPure.logEffect`